doc/pylint.rst

```diff
@@ -1,35 +1,95 @@
-####################
-Static Code Analysis
-####################
+#############
+Code Analysis
+#############
 
-The **Static Code Analysis** pane detects style issues, bad practices, potential bugs, and other quality problems in your code, all without having to actually execute it.
-Spyder's static analyzer is powered by the best in class `Pylint`_ back-end, which can intelligently detect an enormous and customizable range of problem signatures.
+The **Code Analysis** pane detects style issues, bad practices, potential bugs, and other quality problems in your code, all without having to actually execute it.
+Based on this, it gives you a global evaluation of your code reflected on a score from 0 to 10.
+Spyder's code analyzer is powered by the best-in-class `Pylint`_ back-end, which can intelligently detect an enormous and customizable range of problem signatures.
 
 .. _Pylint: https://www.pylint.org/
 
-.. image:: images/pylint/static_analysis_standard.png
+.. image:: images/pylint/code_analysis_standard.png
    :alt: Spyder Pylint pane, showing numerous issues discovered in a file
 
 |
 
 
-=========================
-Using the static analyzer
-=========================
 
-You can select the desired file to analyze directly in the :doc:`editor` by clicking anywhere within it, or you can manually enter the Python module or package path you'd like it to check in the path entry textbox in the :guilabel:`Static Code Analysis` pane.
-The analyzer works with both ``.py`` (or ``.pyw``) Python scripts and whole Python packages (directories containing an :file:`__init__.py` file).
-Then, to run it, select :menuselection:`Sour&ce --> Run static code analysis` from the menubar, click the :guilabel:`Analyze` button in the pane, or press the configurable static analysis shortcut (:kbd:`F8` by default).
-Cancel analyzing a file with the :guilabel:`Stop` button, and if analysis fails, click the :guilabel:`Output` button to find out why.
+=======================
+Using the code analyzer
+=======================
 
-To go directly to the file and line in the :doc:`editor` highlighted by a failed check, just click its name.
-You can click the dropdown or press the :kbd:`Down Arrow` key in the filename field to view results of previous analyses; the number of recent runs Spyder should remember can be customized in the :guilabel:`History` dialog from the :guilabel:`Static Code Analysis` context menu.
+You can select the desired file to analyze directly in the :doc:`editor` by clicking anywhere within it.
+To run the analysis, click the :guilabel:`Analyze` button in the Code Analysis pane, press the configurable shortcut (:kbd:`F8` by default) or select :menuselection:`Source --> Run code analysis` from the menu bar.
```

Comment:
```suggestion
To run the analysis, press the configurable shortcut (:kbd:`F8` by default), select :menuselection:`Source --> Run code analysis` from the menu bar or click the :guilabel:`Analyze` button in the Code Analysis pane (if already visible).
```

This needs to be rephrased because the `Code Analysis` pane is not shown by default the first time you open Spyder.

Reply:
LGTM

Reply:
```suggestion
To run the analysis, press the configurable shortcut (:kbd:`F8` by default), select :menuselection:`Source --> Run code analysis` from the menu bar or click the :guilabel:`Analyze` button in the Code Analysis pane.
If the Code Analysis pane is not visible, you can open it under :menuselection:`View --> Panes --> Code Analysis`.
```